.rubocop.yml

```diff
@@ -1,5 +1,5 @@
 AllCops:
-  TargetRubyVersion: 2.4
+  TargetRubyVersion: 2.7
```

Comment:
Perhaps this should be 2.6, since RuboCop may offer changes that don't work in 2.6 with this setting?

Reply:
Sounds reasonable! Yes, I guess it should be the lowest supported version. 

Are you using JRuby at all? Am I correct in thinking 9.1 is old and not worth building against.

Reply:
Currently not using JRuby, and yes, 9.2 is the currently interesting version. 

Already from this, I think 9.1 should be let go from the CI matrix.

> JRuby 9.1.17.0 represents our stable release of our Ruby 2.3.x support

Reply:
Great - many thanks for the feedback 🙏